src/admin/admin-dashboard-card/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import {
+  Card,
+  CardContent,
+  CardActions,
+  Grid,
+  makeStyles
+} from '@material-ui/core'
+import PropTypes from 'prop-types'
+import { AdminDashboardCardBody } from 'admin/admin-dashboard-card/body'
+import { AdminDashboardCardIcon } from 'admin/admin-dashboard-card/icon'
+import { AdminDashboardLink } from 'admin/admin-dashboard-card/link'
+
+const useStyles = makeStyles({ cardActions: { backgroundColor: '#F9FAFB' } })
+
+export const AdminDashboardCard = ({ icon, title, statistics, link, linkText }) => (
```

Comment:
Would the name `Card` be more relevant here? We can easily omit the material ui naming collisions by importing it like: `import * as Material from '@material-ui/core'.`